src/haddock/libs/libplots.py

```diff
@@ -834,6 +836,13 @@ def clt_table_handler(clt_file, ss_file):
     # table of structures
     structs_df = find_best_struct(ss_file, number_of_struct=10)
 
+    # if the run will be cleaned, the structures are going to be gzipped
+    if is_cleaned:
+        # substitute the values in the df by adding .gz at the end
+        structs_df = structs_df.replace(
+            to_replace=r"(\.pdb)$", value=r".pdb.gz", regex=True
+        )
```

Comment:
I would add a check that the file exists, just to make sure.

Reply:
the problem here is that the gzipped file does not exist when this is executed, but will exist only after the cleaning is performed (later in the workflow)

Reply:
ow, then lets trust